docs/books/admin_guide/05-vi.md

```diff
@@ -136,65 +136,65 @@ The cursor is placed under the desired character.
 
 * Move one or `n` characters to the left:
 
-<kbd>←</kbd>, <kbd>n</kbd><kbd>←</kbd>, <kbd>h</kbd> or <kbd>n</kbd><kbd>h</kbd>
+++left++, ++n++ ++left++, ++h++ or ++n++ ++h++
 
 * Move one or `n` characters to the right:
 
-<kbd>→</kbd>, <kbd>n</kbd><kbd>→</kbd>, <kbd>l</kbd> or <kbd>n</kbd><kbd>l</kbd>
+++right++, ++n++ ++right++, ++l++ or ++n++ ++l++
 
 * Move one or `n` characters up:
 
-<kbd>↑</kbd>, <kbd>n</kbd><kbd>↑</kbd>, <kbd>k</kbd> or <kbd>n</kbd><kbd>k</kbd>
+++up++, ++n++ ++up++, ++k++ or ++n++ ++k++
 
 * Move one or `n` characters down:
 
-<kbd>↓</kbd>, <kbd>n</kbd><kbd>↓</kbd>, <kbd>j</kbd> or <kbd>n</kbd><kbd>j</kbd>
+++down++, ++n++ ++down++, ++j++ or ++n++ ++j++
 
 * Move to the end of the line:
 
-<kbd>$</kbd> or <kbd>END</kbd>
+++$++ or ++END++
 
 * Move to the beginning of the line:
 
-<kbd>0</kbd> or <kbd>POS1</kbd>
+++0++ or ++POS1++
 
 ### From the first character of a word
 
 Words are made up of letters or numbers. Punctuation characters and apostrophes separate words.
 
-If the cursor is in the middle of a word <kbd>w</kbd> moves to the next word, <kbd>b</kbd> moves to the beginning of the word.
+If the cursor is in the middle of a word ++w++ moves to the next word, ++b++ moves to the beginning of the word.
 
 If the line is finished, VI goes automatically to the next line.
 
 * Move one or `n` words to the right:
 
-<kbd>w</kbd> or <kbd>n</kbd><kbd>w</kbd>
+++w++ or ++n++ ++w++
 
 * Move one or `n` words to the left:
 
-<kbd>b</kbd> or <kbd>n</kbd><kbd>b</kbd>
+++b++ or ++n++ ++b++
 
 ### From any location on a line
 
 * Move to last line of text:
 
-<kbd>G</kbd>
+++G++
 
 * Move to line `n`:
 
-<kbd>n</kbd><kbd>G</kbd>
+++n++ ++G++
```

Comment:
I'm assuming that this should be a lower-case "n", and in order to show that exactly, this command would be: 
```
++"n"+"G"++
```